pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "django-allianceutils"
-version = "3.0.0"
+version = "3.1.0"
```

Comment:
don't bump version in the PR. general process is make the changes, update CHANGELOG.MD adding changes under `3.x.x unreleased`

then we'll do a release separately which may include other changes